tests/ballerina-compiler-api-test/src/test/java/io/ballerina/semantic/api/test/allreferences/FindRefsInFailTrtryStmtTest.java

```diff
@@ -64,6 +64,12 @@
                 {87, 21, location(87, 20, 22),
                         List.of(location(87, 20, 22),
                                 location(88, 20, 22))
+                },
+                {104, 20, location(103, 20, 21),
+                        List.of(location(103, 20, 21),
+                                location(104, 20, 21)
+                        )
+
```

Comment:
Do we need this extra line?

Reply:
Removed in 0c8d1ed8a6d5d93eacc14e6b7d5ebafb004dfe49